DDRMenu/DDRMenu.cpp

```diff
@@ -80,7 +80,11 @@ int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, PWSTR pCmdLine
     while(true) {
         io->Tick();
         menu->Tick();
-        display->Tick();
+        
+        if(display->Tick(menu->SecondsLeft()))
+        {
+            menu->ResetTimeout();
```

Comment:
This is going to break booting on P3IO if you run too far past the 30 second timeout. I never figured out why the watchdog pet wasn't working, but if you let this menu run for more than ~40 seconds, then when you get into a game the IO init will end up failing due to the IO board itself locking up. On a real kit (dragon or unicorn), you'll also get a power cycle.

This is why I ended up choosing 30 seconds and not resetting the timeout.